src/plugins/auto/plugin.cpp

```diff
@@ -115,6 +115,10 @@ std::vector<DeviceInformation> MultiDeviceInferencePlugin::ParseMetaDevices(cons
         return "";
     };
 
+    unsigned int devicePriority = 0;
+    auto prioritiesIter = config.find(MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES);
```

Comment:
the PR seems to be about new API, why we need this code?

Reply:
fix it. I only update the key in this PR, need another PR to remove all old keys and replace with new configure 2.0 key in code of auto plugin ,relate test, and document